github/files/drupal7/scripts/devops/reinstall/post_settings.yml

```diff
@@ -23,6 +23,14 @@
   when: item.status == true
   with_items: env.post_settings
 
+- name: Finding a domain module
+  shell: 'find . -name domain -type d | grep contrib | cut -d. -f2 || true'
```

Comment:
It is a not optimum way of checking domain availability
Let's use ```drush pml``` for finding ```domain_access``` module enabled